tests/pytests/unit/states/test_helm.py

```diff
@@ -0,0 +1,269 @@
+import pytest
+import salt.states.helm as helm
+from tests.support.mock import MagicMock, patch
+
+
+@pytest.fixture
+def configure_loader_modules():
+    return {helm: {}}
+
+
+def test_repo_managed_import_failed_repo_manage():
+    ret = {
+        "name": "state_id",
+        "changes": {},
+        "result": False,
+        "comment": "'helm.repo_manage' modules not available on this minion.",
+    }
+    assert helm.repo_managed("state_id") == ret
+
+
+def test_repo_managed_import_failed_repo_update():
+    mock_helm_modules = {"helm.repo_manage": MagicMock(return_value=True)}
+    with patch.dict(helm.__salt__, mock_helm_modules):
+        ret = {
+            "name": "state_id",
+            "changes": {},
+            "result": False,
+            "comment": "'helm.repo_update' modules not available on this minion.",
+        }
+        assert helm.repo_managed("state_id") == ret
+
+
+def test_repo_managed_is_testing():
+    mock_helm_modules = {
+        "helm.repo_manage": MagicMock(return_value=True),
+        "helm.repo_update": MagicMock(return_value=True),
+    }
```

Comment:
**suggestion(non-blocking)** these could go in the `configure_loader_modules`, and `ret` looks like it could be a series of fixtures, e.g.

```
@pytest.fixture
def failed_result():
    return {"name": "state_id", "result": False, "changes": {}}
    
@pytest.fixture
def failed_update_result(failed_result):
    failed_result["comment"] = "'helm.repo_update' modules not available on this minion."
    return failed_result
```

Extracting setup into fixtures allows the tests to be much more focused (and provides opportunity for reuse)